dotnet/src/AutoGen.SemanticKernel/SemanticKernelAgent.cs

```diff
@@ -27,41 +27,22 @@ namespace AutoGen.SemanticKernel;
 /// 
 /// <para>To support more AutoGen built-in <see cref="IMessage"/>, register with <see cref="SemanticKernelChatMessageContentConnector"/>.</para>
 /// </summary>
-public class SemanticKernelAgent : IStreamingAgent
+public class SemanticKernelAgent(
+    ChatCompletionAgent chatCompletionAgent,
+    string name,
+    string systemMessage = "You are a helpful AI assistant")
+    : IStreamingAgent
 {
-    private readonly Kernel _kernel;
-    private readonly string _systemMessage;
-    private readonly PromptExecutionSettings? _settings;
-
-    public SemanticKernelAgent(
```

Comment:
Can you keep this constructor so it won't introduce breaking change?

Reply:
I added back this constructor. Could you double check to see if this is what you expected? Kernel is removed as a private property since there is no longer any use.

Reply:
LGTM